test/main.spec.js

```diff
@@ -0,0 +1,14 @@
+import test from 'tape'
+import addTwoNumbers from './../src/main'
```

Comment:
Yes (sorry I forgot about that), just in the package.json of the application itself, not in the eslint config.